src/sass/components/_grid.scss

```diff
@@ -88,7 +86,7 @@ $gutter: $md;
   }
 
   &--tight {
-    margin: 0 ($gutter/-2);
+    margin: 0 ($gutter/-1);
   }
```

Comment:
Right now this `--tight` modifier has no affect since you're dividing by negative on (`-1`). You'll need to divide by something less than one to make the gutters smaller which is the intended affect of the `--tight` row modifier. Maybe `.75` to start with?